centreon/packages/ui/src/ActionsList/index.tsx

```diff
@@ -28,15 +29,15 @@ const useStyles = makeStyles()({
   }
 });
 
-const ActionsList = ({ className, actions }: Props): JSX.Element => {
+const ActionsList = ({ className, actions, onClick }: Props): JSX.Element => {
   const { cx, classes } = useStyles();
   const { t } = useTranslation();
 
   return (
-    <MenuList className={cx(classes.list, className)}>
-      {actions?.map(({ Icon, label, onClick }) => {
+    <MenuList className={cx(classes.list, className)} onClick={onClick}>
```

Comment:
Why do you want a click listener on MenuList?

Reply:
there is an overflow between the two popups. I want to close the action popup while the confirm/duplicate popup is shown

Reply:
Ok but it's related to the Menu component

Reply:
the prop onClick can be used for any action

Reply:
In you can, you want to close the popup while the modal is displaying, right?

Reply:
yes

Reply:
Ok so you don't need the onClick. You just have to set a state that displays the modal and fades away the Menu popup

Reply:
yes, you are right